### PR TITLE
Add list item to increase accessibilty

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Uses ul and li html tags for comments

## Why?
Increase accessibility for screen readers

## Link to supporting Trello card
https://trello.com/c/zmOLaFp7/1270-accessibility-use-ul-li-for-comment-list-and-top-picks-list

### CODE REVIEW

File: src/components/CommentContainer/CommentContainer.tsx [hunk 1]
```diff
@@ -129,15 +134,19 @@ export const CommentContainer = ({
...
-                pillar={pillar}
-                setCommentBeingRepliedTo={setCommentBeingRepliedTo}
-                user={user}
-                isReply={true}
-              />
-            ))}
+            <ul className={commentContainerStyles}>
+              {responses.map(responseComment => (
+                <li>
+                  <Comment
```
Comment (oliverlloyd) [inline src/components/CommentContainer/CommentContainer.tsx]: As above, how do you feel about making `Comment` an `li`?

File: src/App.tsx [hunk 2]
```diff
@@ -286,17 +288,19 @@ export const App = ({ shortUrl, user }: Props) => {
...
-              shortUrl={shortUrl}
-              onAddComment={onAddComment}
-              user={user}
-              threads={filters.threads}
-              commentBeingRepliedTo={commentBeingRepliedTo}
-              setCommentBeingRepliedTo={setCommentBeingRepliedTo}
-            />
+            <li>
+              <CommentContainer
+                key={comment.id}
```
Comment (oliverlloyd) [inline src/App.tsx]: Another reason to not have the wrapper is this `key` prop won't need to be lifted

File: src/components/CommentContainer/CommentContainer.tsx [hunk 3]
```diff
@@ -117,7 +122,7 @@ export const CommentContainer = ({
   };
 
   return (
-    <div>
+    <>
```
[the remark this answers is not in the document]
Comment (liywjl) [reply src/components/CommentContainer/CommentContainer.tsx]: I think it helps to reason with when the `ul` and `li` are in the same file, sure we can remove a fragment, but I feel that a fragment here isnt much of an issue in accessibility/performance/readability. To the contrary I think it improves readability

File: src/App.tsx (hunk 2 above)
Comment (liywjl) [reply src/App.tsx]: good catch :)

File: src/components/CommentContainer/CommentContainer.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (oliverlloyd) [reply src/components/CommentContainer/CommentContainer.tsx]: Fair enough. Happy to merge 👍

**Pull request merged**